### PR TITLE
Cleanup zetachain repo to be zeta apps repo

### PR DESCRIPTION
**Pull request by andresaiello**
### Summary
- Remove address pkg
- Remove address-tools pkg
- Remove examples pkg
- Update all the references to use protocol module addresses

### CODE REVIEW

**Review by lucas-janon — DISMISSED** [review]
Nice one 👌

File: packages/zeta-app-contracts/scripts/address.helpers.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { networks } from "@zetachain/networks";
+import { isProtocolNetworkName, ZetaProtocolNetwork } from "@zetachain/protocol-contracts";
+import { readFileSync, writeFileSync } from "fs";
+import { network } from "hardhat";
+import { join } from "path";
+
+import addresses from "../data/addresses.json";
+
+export const getAppAddress = (address: string, network: ZetaProtocolNetwork): string => {
+  return (addresses["ccm"] as any)[network][address];
```
Comment (lucas-janon) [inline packages/zeta-app-contracts/scripts/address.helpers.ts:10]: I think it would be cool to type this one.

Comment (andresaiello) [reply packages/zeta-app-contracts/scripts/address.helpers.ts:12]: done

**Review by lucas-janon — APPROVED** [review]
LGTM!

**Pull request merged**